enjoy.py

```diff
@@ -101,6 +102,12 @@ def main():  # noqa: C901
         checkpoints = glob.glob(os.path.join(log_path, "rl_model_*_steps.zip"))
         if len(checkpoints) == 0:
             raise ValueError(f"No checkpoint found for {algo} on {env_id}, path: {log_path}")
+
+        def natsort(s):
+            # we need to sort the list in natural order, but there is no standard python function for it
+            return [int(t) if t.isdigit() else t.lower() for t in re.split(r"(\d+)", s)]
```

Comment:
why not something more specific to this use case?
I'm thinking of `int(checkpoint_path.split("_")[2]) # path follow the pattern "rl_model_*_steps.zip"`

Reply:
Yeah I just thought about sorting it in natural order, not that there is a simpler solution in this special case. Your proposed solution would also work, as long as the naming scheme does not change. But this is highly unlikely. I can change this, if you want.

Reply:
Your solution is also better to understand for any reader, so I guess it would be better.